internal/resources/examples_test.go

```diff
@@ -82,6 +82,12 @@ func TestAccExamples(t *testing.T) {
 				testutils.CheckCloudInstanceTestsEnabled(t)
 			},
 		},
+		{
+			category: "Connections",
+			testCheck: func(t *testing.T, filename string) {
+				t.Skip() // TODO: Make all examples work
```

Comment:
This is copied from the other implementations: this is re-running the acceptance tests here in TestAccExamples, but without the declaration of the mock httptest server, these tests fail. I'm not sure what the long-term solution plan is.

The acceptance tests still run and are passing elsewhere.